save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt

```diff
@@ -309,7 +313,8 @@ class ExecutionView : AbstractView<ExecutionProps, ExecutionState>(Style.SAVE_LI
     } ?: ""
 
     private suspend fun getAdditionalInfoFor(testExecution: TestExecutionDto, id: String) {
-        val trDebugInfoResponse = getDebugInfoFor(testExecution)
+        val trDebugInfoResponse = getDebugInfoFor(testExecution.requiredId())
+        // FixMe: invalid setup of execution because of the invalid propagated ID
```

Comment:
@nulls this does not work. Looks like it is the one of the reasons why we don't see logs for INTERNAL_ERROR 

I fixed debug info, but this part with execution info was lost while we were migrating to S3.